### PR TITLE
feat(web): feature/BOAT-851-disable-change-journey

### PR DESCRIPTION
**Pull request by kandcdev**
## Describe your changes
BOAT-851

disable change start date journey as this will be addressed in a separate ticket

## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [x] I have performed a self-review of my own code
- [x] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [x] I have provided details on how I have tested my code
- [x] I have referenced the ticket number above
- [x] I have provided a description of the ticket
- [x] I have included unit tests to cover any testable code changes

### CODE REVIEW

**Review by glpins — APPROVED** [review]

File: appeals/web/src/server/appeals/appeal-details/__tests__/__snapshots__/appeal-details.test.js.snap [hunk 1]
```diff
@@ -360,7 +360,7 @@ exports[`appeal-details GET /:appealId should render a Decision inset panel when
                         </div>
                         <div class="govuk-summary-list__row"><dt class="govuk-summary-list__key"> Start date</dt>
                             <dd class="govuk-summary-list__value">23 May 2023</dd>
-                            <dd class="govuk-summary-list__actions"><a class="govuk-link" href="/appeals-service/appeal-details/2/start-case/date"> Change<span class="govuk-visually-hidden"> The date the case was started</span></a>
+                            <dd class="govuk-summary-list__actions"><a class="govuk-link" href="/appeals-service/appeal-details/2/#"> Change</a>
```
Comment (AlexMoulin) [inline appeals/web/src/server/appeals/appeal-details/__tests__/__snapshots__/appeal-details.test.js.snap:363]: We've gone from a complete url to a url with # in it?

**Review by AlexMoulin — CHANGES_REQUESTED** [review]
Reviewing

File: appeals/web/src/server/appeals/appeal-details/__tests__/__snapshots__/appeal-details.test.js.snap (hunk 1 above)
Comment (AlexMoulin) [reply appeals/web/src/server/appeals/appeal-details/__tests__/__snapshots__/appeal-details.test.js.snap:363]: If this is tied to the fact that change is another ticket then we probably need a TODO and link the ticket to BOAT851

**Review by AlexMoulin — CHANGES_REQUESTED** [review]

**Review by AlexMoulin — APPROVED** [review]
Discussed and found the ticket for the follow up work. All is good :)

**Pull request merged**